src/main/java/sg/ncl/MainController.java

```diff
@@ -2085,6 +2081,7 @@ private boolean isNotAdminAndNotInTeam(HttpSession session, Realization realizat
     }
 
     //---------------------------------Dataset Page--------------------------
+    private static final String CONTRIBUTE_DATA_PAGE = "data_contribute";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1213)